app/assets/stylesheets/application.css.scss

```diff
@@ -24,7 +24,7 @@
 @import 'foundation-icons';
 .background-image {
   background: image-url('background_picture.jpg');
-  padding: 195px 5px 195px 5px;
+  padding: 170px 5px 170px 5px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `170px 5px` instead of `170px 5px 170px 5px`